src/pages/Rocket.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Rockets = () => (
+  <div>
+    <p>Rockets</p>
+  </div>
+);
+
+export default Rockets;
```

Comment:
- [ ] You have this extra module that you are not using. Please delete this file. 💪 👏